### DISCUSSION TITLE
The MapLibre Technical Steering Committee Meeting - August 9th, 2023 (MapLibre GL JS)

### DISCUSSION THREAD

**Discussion by duje** [maplibre/maplibre#307] [Technical Steering Committee Meetings]
As always 8 to 9PM CEST, email invite sent August 2nd, 2023. In case you are still missing your invitation, ping me on Slack: https://maplibre.org/about/duje/.

**Comment by wipfli** [comment]
I think it is CEST, right?

**Comment by duje** [comment]
Attendees:

* Harel Mazor @HarelM Israel Hiking Map/MapLibre board member/Web maintainer
* Luke Seelenbinder, @lseelenbinder, MapLibre / Stadia Maps
* Isaac Besora @ibesora Felt
* Duje Schröder @duje, MapLibre Coordinator Assistant
* Yuri Astrakhan @nyurik, MapLibre & Rivian
* Oliver Wipfli, @wipfli, MapLibre Coordinator
* Bart Louwers @louwers, Independent / MapLibre Native Maintainer
* Craig O’Connor, @CraigglesO, MapTiler
* Petr Pridal, @klokan, MapTiler
* Minh Nguyễn @1ec5, OpenHistoricalMap / OSM Americana
* Xiaoji Chen @pessimistress deck.gl + react-map-gl maintainer, OpenJS Foundation
* Jonah Adkins @jonahadkins Meta

**Comment by duje** [comment]
Topics:

* [HarelM] Bring in contour generation as a new type of source [onthegomap/maplibre-contour: Render contour lines from raster DEM tiles in maplibre-gl-js (github.com)](https://github.com/onthegomap/maplibre-contour) - Keep it as plugin to reduce package size.
   * PMTiles?- Add to examples
* [HarelM] Web components short demo and discussion: https://stackblitz.com/edit/typescript-maplibre-web-component-poc
   * If someone is interested in pushing it forward please ping me
* [HarelM] License - present logo/attribution by default?
* [Wipfli] Status of benchmarks is still not great
